src/usvfs_dll/hookcontext.cpp

```diff
@@ -216,6 +216,56 @@ BOOL HookContext::executableBlacklisted(LPCWSTR wapp, LPCWSTR wcmd) const
   return m_Parameters->executableBlacklisted(app, cmd);
 }
 
+void usvfs::HookContext::addSkipFileSuffix(const std::wstring& fileSuffix) 
+{
+  const auto fsuffix = shared::string_cast<std::string>(fileSuffix, shared::CodePage::UTF8);
+
+  if (fsuffix.empty()) {
+    return;
+  }
+
+  spdlog::get("usvfs")->debug("skipping files with the suffix '{}'", fsuffix);
```

Comment:
This log (and the one for addSkipDirectory) is very misleading - We are not actually skipping files/directories here, we are registering them to be skipped.

Reply:
I tried to follow the same way blacklisted execs are logged, I'll re-do these logs to give them more context